code/modules/projectiles/guns/rifles.dm

```diff
@@ -176,7 +176,7 @@
 	fire_delay = 1 SECONDS
 	accuracy_mult = 1.25
 	scatter = -15
-	burst_amount = 0
+	burst_amount = 1
```

Comment:
If the parent has a burst amount of `1`, then remove the line. Same for the rest.

Reply:
parent has 3; their granddad has 1

Reply:
Then it's fair.